### DISCUSSION TITLE
how to setup timewarp/nightlapse/timelapse/star trails/car lights presets via LABS?

### DISCUSSION THREAD

**Discussion by artstylee** [gopro/labs#460] [Q&A] [answered]
subject
how to setup these presets? as i understand
timewarp/timelapse/nightlapse/star trails are different presets
timewarp has speed (i.e. x10) but no shutter/intervals/trail length
timelapse has intervals but no speed/shutter/trail length
nightlapse has shutter and intervals but no speed/trail lights
trail lights has shutter/trail light but no intervals or speed

how to setup these presets via labs?

**Comment by dnewman-gpsw** [comment, marked as answer]
Some of the new features like trails and not fully configured with Labs alone.  Set them up via menu or Quik app, as the settings are remembered.
